README.md

```diff
@@ -76,9 +76,10 @@ Call this function on figwheel reload.
 - Let the route data decide what view to display
 ```clojure
 (defn main-view []
-  [k/switch-route (fn [route] (:handler route))
+  [k/switch-route (comp :name :data)
```

Comment:
I think it would be simpler for newcomers to use normal `fn` syntax, as `comp` can be a bit alien to some.